test/server/cards/07-GR/RoragScreamer.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('Rorag Screamer', function () {
+    describe("Rorag Screamer's ability", function () {
+        beforeEach(function () {
+            this.setupTest({
+                player1: {
+                    house: 'brobnar',
+                    hand: ['rorag-screamer'],
+                    inPlay: ['cpo-zytar', 'alaka']
+                },
+                player2: {
+                    inPlay: ['dew-faerie', 'cornicen-octavia']
+                }
+            });
+        });
+
+        it('readies after fighting', function () {
+            this.player1.playCreature(this.roragScreamer);
+            this.player1.endTurn();
+            this.player2.clickPrompt('untamed');
+            this.player2.endTurn();
+            this.player1.clickPrompt('brobnar');
+            this.player1.fightWith(this.roragScreamer, this.dewFaerie);
+            expect(this.roragScreamer.exhausted).toBe(false);
+            this.player1.fightWith(this.roragScreamer, this.dewFaerie);
+            expect(this.roragScreamer.exhausted).toBe(false);
+            this.player1.fightWith(this.roragScreamer, this.cornicenOctavia);
+            expect(this.roragScreamer.location).toBe('discard');
+        });
+
+        it('can make a creature lose elusive for the turn', function () {
+            this.player1.clickCard(this.roragScreamer);
```

Comment:
as a suggestion, add a `scrap` method, this will make development simpler.

Reply:
Good idea, added and updated all existing tests,